tst/com/amazon/corretto/crypto/provider/test/EvpSignatureTest.java

```diff
@@ -63,7 +54,7 @@
             tmpMap.put("RSA", kg.generateKeyPair());
 
             kg = KeyPairGenerator.getInstance("EC");
-            kg.initialize(new ECGenParameterSpec("NIST P-384"));
+            kg.initialize(new ECGenParameterSpec("NIST P-521"));
```

Comment:
Is the change from P-384 to P-521 intentional as part of this PR, or an artifact left over from local testing? (It looks harmless either way.)

Reply:
This is intentional. P-521 is large enough that the DER encoded signature needs a "definite, long" length encoding rather than a single-byte "definite, short" length encoding. By making it larger we cover more edge-cases.

Reply:
That makes perfect sense to me.